README.md

```diff
@@ -186,7 +186,8 @@ scrape_configs:
 | web.telemetry-path                  | REDIS_EXPORTER_WEB_TELEMETRY_PATH                | Path under which to expose metrics, defaults to `/metrics`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| redis-only-metrics                  | REDIS_EXPORTER_REDIS_ONLY_METRICS                | Whether to also export go runtime metrics, defaults to false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| redis-only-metrics                  | REDIS_EXPORTER_REDIS_ONLY_METRICS                | Whether to export only Redis metrics (omit Go runtime metrics), defaults to false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| extended-go-metrics                 | REDIS_EXPORTER_EXTENDED_GO_METRICS               | Whether to export all Go runtime metrics, defaults to false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Naming is hard 😅 

Wdyt if this option renamed to `enable-go-runtime-metrics` or `include-go-runtime-metrics` or `go-runtime-metrics`?

I don't think they're extended. We're basically either collecting redis_exporter **process** metrics, which are technically Go runtime metrics, but they could be called just process metrics.

Reply:
yes! i like that.
you're right, we got Golang runtime metrics, and then we got "process metrics"

I like `include-go-runtime-metrics` - what do you think?

Reply:
Yep, I think it sounds good. 

Changed PR with renamed variables, PTAL when have time.